CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 #ARAnalytics
 
+* Removes TestFlight ( @ashfurrow )
```

Comment:
this is on v3.0.0, I've not been able to release yet, so it will probably be post RAC